### DISCUSSION TITLE
GitHub Copilot always suggest some code when I am typing, is there a way so we can press a key combination to trigger it like ctrl + space in vscode

### DISCUSSION THREAD

**Discussion by Tetrax-10** [community/community#15846] [Copilot Conversations]
it should not display any ghost code or suggest in vscode until I press a key combination

**Comment by Fire-** [comment]
There's an extension setting for the inline suggestions toggle[1], but unfortunately control+enter only brings up the full ten synthesization window rather than triggering the inline results - I'd like to see this as well.

1:
<img width="339" alt="image" src="https://user-images.githubusercontent.com/1456217/166862676-97d1db7b-d77f-47aa-bbe8-3292e8812094.png">

**Comment by jrnkng** [comment]
Did you figure this out? I only want suggestions when I ask for it!

**Reply by Tetrax-10** [reply]
nope

**Comment by isinaltinkaya** [comment]
This is a much needed feature!

Meanwhile, you can try setting a keyboard shortcut to enable/disable the extension altogether, which is not ideal but works fine with settings cycler. See #7553
